main.js

```diff
@@ -176,24 +181,27 @@ const renderFooter = () => {
 };
 
 const pinRepoEvent = () => {
-  document.querySelector("#pinnedRepos").addEventListener("click", (e) => {
+  document.querySelector("#repoForm").addEventListener("click", (e) => {
     if (e.target.id === "repoModal-btn") {
       let repoList = "";
       for (const i of repo) {
         repoList += `<li class="list-group-item">
-        <input id="repo${i.id}" class="form-check-input me-1" type="checkbox" value="..." aria-label="...">
+        <input id="${i.id}" class="form-check-input me-1" name="repos" type="checkbox" value="${i.name}" aria-label="...">
```

Comment:
MIght want to make the `i` variable more descriptive. Usually in code, `i` stands for index.